js/windowMgr.js

```diff
@@ -496,73 +496,76 @@ function doCreateMainWindow(initialUrl, initialBounds) {
      * @param webContents Web contents of the window
      */
     function handlePermissionRequests(webContents) {
-        
+
         let session = webContents.session;
-        
+
         getConfigField('permissions')
             .then((permissions) => {
-                
+
                 if (!permissions) {
                     log.send(logLevels.ERROR, 'permissions configuration is invalid, so, everything will be true by default!');
                     return;
                 }
-                
+
                 session.setPermissionRequestHandler((sessionWebContents, permission, callback) => {
-                    
+
                     function handleSessionPermissions(userPermission, message, cb) {
-                        
+
                         log.send(logLevels.INFO, 'permission is -> ' + userPermission);
-                        
+
                         if (!userPermission) {
                             let fullMessage = `Your administrator has disabled ${message}. Please contact your admin for help.`;
-                            electron.dialog.showMessageBox(BrowserWindow.fromWebContents(webContents), {type: 'error', title: 'Permission Denied!', message: fullMessage});
+                            const browserWindow = BrowserWindow.getFocusedWindow();
+                            if (browserWindow && !browserWindow.isDestroyed()) {
+                                electron.dialog.showMessageBox(browserWindow, {type: 'error', title: 'Permission Denied!', message: fullMessage});
```

Comment:
So, if `browserWindow` is not accessible/available, does that mean we don't show the error message at all?

Reply:
Yes that's correct, We don't show the error message if the `browserWindow` is not available

Reply:
Ok. But, how are we going to communicate to the user that that particular functionality is unavailable to him/her?

Reply:
@VikasShashidhar This only happens when the user starts the meeting and then changes the focus on to a different application, but there is a red banner shown for a few secs 
![screen shot 2018-05-21 at 1 49 22 pm](https://user-images.githubusercontent.com/13243259/40297783-8ecf9bd4-5cfe-11e8-8055-93b5495508b6.png)

The electron dialog will be shown to the user when he/she interacts with any of the actions buttons (In this case the `browserWindow` will always be defined)